### ISSUE TITLE
Keep crashing: JSONDecodeError: Expecting property name enclosed in double quotes

### ISSUE THREAD

**Issue by Extarys** [kingstefan26/alabamaEncoder#8]
I put all the episodes inside a folder and run a bash script that convert the video then use mvkmerge to merge in subtitles and move the file to my NAS. But for some reason, the transcoding process now crashes at every file.

```
WORKERS 6 CPU 92% MEM 29% ESTM BITRATE 1980.30 kb/s:  95%|██████████████▏| 36.4k/38.4k [2:07:54<07:52, 4.25frame/s]
Traceback (most recent call last):
  File "/home/user/.local/bin/alabamaEncoder", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/user/alabamaEncoder/cli/__main__.py", line 155, in main
    asyncio.run(job.run_pipeline())  # this runs the whole encoding process
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/user/alabamaEncoder/core/job.py", line 246, in run_pipeline
    await asyncio.create_task(
  File "/home/user/alabamaEncoder/parallel_execution/execute_commands.py", line 206, in execute_commands
    rslt = await future
           ^^^^^^^^^^^^
  File "/usr/lib64/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/alabamaEncoder/core/chunk_job.py", line 96, in run
    self.ctx.get_kv().set("chunk_timing", self.chunk.chunk_index, timing_stats)
  File "/home/user/alabamaEncoder/core/util/kv.py", line 40, in set
    bucket_content = self._load(bucket)
                     ^^^^^^^^^^^^^^^^^^
  File "/home/user/alabamaEncoder/core/util/kv.py", line 67, in _load
    return json.load(f)
           ^^^^^^^^^^^^
  File "/usr/lib64/python3.12/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
               ^^^^^^^^^^^^^^^^^^^^^^
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 40926 (char 40925)
Current Session Runtime: 2 hour(s) 601 second(s), Runtime From Previous Sessions: 0 second(s), Total Runtime: 2 hour(s) 601 second(s)
WORKERS 6 CPU 92% MEM 29% ESTM BITRATE 1980.30 kb/s:  95%|█████████▍| 36.4k/38.4k [2:09:02<07:06, 4.70frame/s]
```

Relaunch after crash:
```
 ./alabama_start.sh

==============================================================================================
=> File is ./Season01/S01 E01.mkv
=> Episode is: S01 E01
==============================================================================================
Checking if transcoding is needed
> Starting transcode
Input Video: 1920x1080 @ 29.97 fps, YUV420P, SDR, 6976.34 Kb/s, H264
Using SVT_AV1 version: SVT-AV1-PSY v2.1.0-A (release)
PSY Release:
Using dynamic vmaf targeting
Checking files: 100%|██████████████████████████████████████████████████████| 519/519 [00:00<00:00, 13874.79file/s]
Starting encoding of 23 out of 519 scenes
WORKERS 4 CPU 57% MEM 31% ESTM BITRATE 1986.14 kb/s:   0%|                        | 0.00/1.58k [00:28<?, ?frame/s]
...
WORKERS 6 CPU 81% MEM 32% ESTM BITRATE 1994.03 kb/s:  62%|███████████       | 972/1.58k [03:51<02:06, 4.79frame/s]

Checking files: 100%|████████████████████████████████████████████████████| 519/519 [00:00<00:00, 1688784.93file/s]
All chunks passed integrity checks 🤓
Concating Video
size=  310956kB time=00:21:21.98 bitrate=1987.0kbits/s speed= 165x    
Skipping audio
size=  310699kB time=00:21:21.98 bitrate=1985.4kbits/s speed=2.63e+03x    
Concat took 10.430099964141846 seconds
Traceback (most recent call last):
  File "/home/jer/.local/bin/alabamaEncoder", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/user/alabamaEncode/cli/__main__.py", line 155, in main
    asyncio.run(job.run_pipeline())  # this runs the whole encoding process
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/user/alabamaEncode/core/job.py", line 311, in run_pipeline
    print_stats(
  File "/home/user/alabamaEncode/core/final_touches.py", line 85, in print_stats
    audio_codec = audio_tracks[0]["codec_name"].upper()
                  ~~~~~~~~~~~~^^^
IndexError: list index out of range
Current Session Runtime: 406 second(s), Runtime From Previous Sessions: 2 hour(s) 601 second(s), Total Runtime: 2 hour(s) 1007 second(s)

> Merging
Merge complete
=> Cleaning up
Done
```

When I'm unlucky (90% of the time), I need to delete the temp folder and redo the transcode. I noticed that when there is a space in the folder name for the source, it crashes more often, like `./My Series/Season 01/...`, hence the `Season01` path above.

Transcoding command:
```
FULLPATH="./Season 01/S01E01.mkv"
BASE="./Season 01"
EP="S01E01"

alabamaEncoder encode "${FULLPATH}" "${BASE}/Converted/${EP}.webm" --title "Stat - ${EP}" --vmaf_target 96 --chunk_order even -previews -stats --tune balanced --grain 8 --vmaf_probe_speed 7 --dynamic_vmaf_target --dont_encode_audio \
        --video_filters "unsharp=7:7:0.8:7:7:0.8" --encoder_speed_override 4 --encoder_flag_override "--film-grain-denoise 0 --film-grain 6 --qm-min 0 --qm-max 8 --scd 0 --enable-dlf 2 --enable-variance-boost 1 --sharpness 1 --frame-luma-bias 60"
```

Let me know how I can help you find the cause of this problem ;)

**Comment by kingstefan26** [comment]
Know exactly the source of the first issue, chunk timing stats are saved to one json via my kv, which is prone to timing issues cuz multithreading, ussly doesn't happen cuz GIL but is not impossible 

Second one is a simple if missing

**Comment by kingstefan26** [comment]
just fixed the first issue in 1a4a244d
the second issue was aleady fixed in 0eda21a6 so please update
